src/data/list/cycle.lean

```diff
@@ -667,24 +674,30 @@ fintype.subtype (((finset.univ : finset {s : cycle α // s.nodup}).map
 def to_finset (s : cycle α) : finset α :=
 s.to_multiset.to_finset
 
+@[simp] theorem to_finset_def (s : cycle α) : s.to_multiset.to_finset = s.to_finset :=
```

Comment:
This lemma should get a different name.

Reply:
Would `to_finset_to_multiset` be any better?